src/utils/SnapshotProvider.js

```diff
@@ -58,7 +58,7 @@ const getProposals = async first => {
     { headers: { "Content-Type": "application/json" } }
   );
 
-  return data.data.proposals;
+  return data.data.proposals.filter(p => p.title !== "ee");

```

Comment:
filter by proposal address if possible 0xaCa39B187352D9805DECEd6E73A3d72ABf86E7A0

Reply:
I tried that, but didn't find the way to filter by proposal address.
![image](https://user-images.githubusercontent.com/17890146/122995755-3b01d480-d367-11eb-83dd-2476beb11a0f.png)
![image](https://user-images.githubusercontent.com/17890146/122995821-51a82b80-d367-11eb-96e1-11acf5413574.png)

Reply:
@kesar ProposalWhere custom type doesn't have the option to filter by proposal details

Reply:
https://hub.snapshot.page/graphql?operationName=Proposals&query=query%20Proposals%20%7B%0A%20%20proposals%20(%0A%20%20%20%20first%3A%2020%2C%0A%20%20%20%20skip%3A%200%2C%0A%20%20%20%20where%3A%20%7B%0A%20%20%20%20%20%20space_in%3A%20%5B%22everipediaiq.eth%22%5D%0A%20%20%20%20%7D%2C%0A%20%20%20%20orderBy%3A%20%22created%22%2C%0A%20%20%20%20orderDirection%3A%20desc%0A%20%20)%20%7B%0A%20%20%20%20id%0A%20%20%20%20title%0A%20%20%20%20choices%0A%20%20%20%20start%0A%20%20%20%20end%0A%20%20%20%20snapshot%0A%20%20%20%20state%0A%20%20%20%20author%0A%20%20%20%20space%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20name%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D

Reply:
```
query Proposals {
  proposals (
    first: 20,
    skip: 0,
    where: {
      space_in: ["everipediaiq.eth"],
      author_in: ["0xaCa39B187352D9805DECEd6E73A3d72ABf86E7A0"]
    },
    orderBy: "created",
    orderDirection: desc
  ) {
    id
    title
    choices
    start
    end
    snapshot
    state
    author
    space {
      id
      name
    }
  }
}
```

Reply:
lol @kesar ahaha. Thank you so much